.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] It is reuired that the gitignore has two files ignored. The node_modules and test.md. Kindly create a test.md and ignore it as you did for node_modules.
![image](https://user-images.githubusercontent.com/86200356/192070344-e4ae4456-3403-4c14-9062-333627778e98.png)